e2e/Enclave.mk

```diff
@@ -52,27 +49,20 @@ endif
 
 SGX_COMMON_CFLAGS += -fstack-protector
 
-CARGO_FEATURES = --features=default
```

Comment:
@junichi-tanaka  Can you also remove `CARGO_FEATURES` reference in `enclave` target?

Reply:
@bluele removed in https://github.com/datachainlab/besu-qbft-elc/pull/8/commits/70106c3b5f83db74958ed86e6ce19e66771d5188